RPG-game-Hung/classes/game.py

```diff
@@ -0,0 +1,187 @@
+import random
+import time
+
+from termcolor import cprint, colored
+
+from classes.colors import Color
+from classes.db import Database, not_number
+from classes.actions import Action
+from classes.person import Person
+from classes.items import Item
+from classes.magics import Magic
+
+
+
+class Game:
```

Comment:
Same as `Action` class,
1. `DocString`
2. The use of `@staticmethod` as well as `@classmethod` in all of your classes should be reconsidered. Maybe read more about them [here](https://realpython.com/instance-class-and-static-methods-demystified/#class-methods)